test/Auth/authSaga.js

```diff
@@ -0,0 +1,17 @@
+import test from 'ava';
+import { take, select } from 'redux-saga/effects';
+import { LOGIN_REQUEST } from '../../src/Auth/authModule';
+import loginFlow from '../../src/Auth/authSaga';
+
+test('loginFlow', (t) => {
```

Comment:
`` t `` is not descriptive at all. Is it a convention to use `` t `` when you use ava?